core/src/main/resources/openapi/overview.html

```
@@ -89,7 +89,7 @@ <h1 class="title">APIs</h1>
         <ul>
             <% for (server in api.value.api.servers) { %>
             <li>
-                <%= com.predic8.membrane.core.openapi.util.UriUtil.getPathFromURL(uriFactory,server.url) %><br>
+                <%= api.value.spec.rewrite.basePath ?: com.predic8.membrane.core.openapi.util.UriUtil.getPathFromURL(uriFactory,server.url) %><br>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)